app/webroot/js/wall.reply.js

```diff
@@ -84,73 +84,64 @@ $(document).ready(function(){
-        // alert ("already exist") ; 
-            // we remove the previous inside reply form 
+        // alert ("already exist") ;
+            // we remove the previous inside reply form
             $("#replyFormDiv_" + previousReplyFormInMessageID ).remove();
-            previousReplyFormInMessageID = -1 ;           
+            previousReplyFormInMessageID = -1 ;
         }
 
         if ( hasClickOnReply) {
           //  alert ("has click on reply");
             var currentMessageBody = $('#messageBody_' + currentMessageId );
 
-
-            // replace "reply"  by "close"  
+            // replace "reply"  by "close"
             messageToReplyTo.attr("class" ,"closeLink " + currentMessageId );
             // i know that's a bit "hacky" to retrieve the send message form
-            // but that way we're sure to always have a coherent form, and 
+            // but that way we're sure to always have a coherent form, and
             // we only need to change the helper
             var sendMessageForm = $('#WallSaveForm').clone();
-            
-            // change the form in order to make it unique 
-            sendMessageForm.attr("id" , "replyForm_" + currentMessageId  ); 
+
+            // change the form in order to make it unique
+            sendMessageForm.attr("id" , "replyForm_" + currentMessageId  );
             sendMessageForm.attr("method","");
             sendMessageForm.attr("action","");
             sendMessageForm.find('.submit').addClass("ajaxSubmit");
-            
+
             // note to myself
             // use append instead of  .html(previous + "") because the previous code will be unbound by jquery
-            // as it will be considered as new code 
-            currentMessageBody.append( 
+            // as it will be considered as new code
+            currentMessageBody.append(
                  "<div id=\"replyFormDiv_"+currentMessageId+"\" class=\"message form\" >"
                 + sendMessageForm.html()
                 +"</div>" );
             $.scrollTo('#replyFormDiv_' + currentMessageId, 800, {offset: -100});
-            previousReplyFormInMessageID = currentMessageId ; 
-            
+            previousReplyFormInMessageID = currentMessageId ;
+
             // focus on textarea
             $("#replyFormDiv_"+currentMessageId+" fieldset div textarea").focus();
         }
-
     }
 
-
-    $(document).on('click', '.ajaxSubmit',
-        function(){
-            saveMessage() ;
-            // this line is not in save message due to ajax's asynchronousity
-            // that way we're sure replyFormInMessageId will not be set to -1
-            // before the end of saveMessage
-    }); 
-
+    $('.ajaxSubmit').click(function(){
+        saveMessage() ;
+        // this line is not in save message due to ajax's asynchronousity
+        // that way we're sure replyFormInMessageId will not be set to -1
+        // before the end of saveMessage
+    });
```

Comment:
For some reason the "Save" button (in reply form on the Wall) doesn't work anymore with this change.

Same issue with the "Cancel" button.